assets/onboarding/navigation/index.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect, useMemo } from '@wordpress/element';
+import { Stepper } from '@woocommerce/components';
+import { get, uniq } from 'lodash';
+
+import { useQueryStringRouter } from '../query-string-router';
+
+/**
+ * Merge the navigation state into the steps.
+ * Add isComplete and onClick - when visited.
+ *
+ * @param {Array}    steps        Steps list.
```

Comment:
Could we document the expected contents of this array? I.e. it looks like a `key` parameter is expected in each element, but that is not obvious from the documentation.

Reply:
Sure! My bad! https://github.com/Automattic/sensei/commit/56b4015f2c50f4b9c46ea9f06c94f8189edcc03d